templates/server.conf.j2

```diff
@@ -73,6 +81,11 @@ topology {{ openvpn_topology }}
 # 10.8.0.1. Comment this line out if you are ethernet bridging. See the man
 # page for more info.
 server {{ openvpn_server }}
+{% if openvpn_ipv6_enabled and openvpn_ipv6_server is defined %}
+server-ipv6 {{ openvpn_ipv6_server }}
+ifconfig-ipv6 {{ openvpn_ipv6_ifconfig }}
+push "route-ipv6-default {{ openvpn_ipv6_route_default }}"
```

Comment:
I couldn't find an option named `route-ipv6-default`. Where is this documented?

Also, why do you need lines 86-87 unconditionally when ipv6 is used?

Reply:
`openvpn_ipv6_server is defined` will always be true. This variable is defined in defaults/main.yml as an empty string, what will matter in this template is if it's still empty or not.

According to the [OpenVPN wiki](https://community.openvpn.net/openvpn/wiki/IPv6#Configstanzawithexpandeddirectives)
>  There are 2 ways to add IPv6 addressing and pool options to the server, similar to what OpenVPN supports for IPv4: using a helper-directive, and by expanding the helper-directive. The expansion is required if you do not wish to use the automatic values the helper-directive supplies.

* therefore `ifconfig-ipv6` will be a complementary and optional configuration
* `route-ipv6-default` doesn't exist, it's optional, and can be pushed with the variable `openvpn_ipv6_route_ranges` which is defined as an empty list by default.

Reply:
Done in c1cbc8c